enclave-tls/src/crypto_wrappers/opencrypt/cleanup.c

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2020-2021 Alibaba Cloud and Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+#include <enclave-tls/log.h>
+#include <enclave-tls/crypto_wrapper.h>
+#include "opencrypt.h"
+
+crypto_wrapper_err_t opencrypt_cleanup(crypto_wrapper_ctx_t *ctx)
+{
```

Comment:
Please refer to wolfssl instance to add the debugging codes in each method, such as `ETLS_DEBUG("ctx %p\n", ctx);`.